website/docs/introduction.md

```diff
@@ -19,13 +19,13 @@
 
 ## Available APIs
 
-- [E-Commerce](api/apps/e-commerce)
-- [Video Library](api/apps/video-library)
-- [Social Media](api/apps/social-media)
-- [Notes App](api/apps/notes-app)
-- [Forum App](api/apps/forum-app)
-- [Project Management](api/apps/project-management)
-- [Habit Tracker](api/apps/habit-tracker)
+- [E-Commerce](apps/e-commerce)
```

Comment:
Hi @sashaboi , seems the website build is failing (it is related to the broken links in docusaurus).
To reproduce this error, run the following steps:
1. cd website
2. yarn install
3. yarn build
4. yarn start

It will throw error on third step. Rectify and when build is successful, push the changes. Thanks for the PR, we can merge it after that.